### PR TITLE
Default retention policy

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

File: monasca_persister/conf/influxdb.py [hunk 1]
```diff
@@ -20,6 +20,12 @@
     cfg.StrOpt('database_name',
                help='database name where metrics are stored',
                default='mon'),
+    cfg.BoolOpt('db_per_tenant',
+                help='Whether to use a separate database per tenant',
+                default=False),
+    cfg.BoolOpt('default_retention_policy',
```
Comment (dougszumski) [inline monasca_persister/conf/influxdb.py]: I added some questions on this upstream - please see those.

**Review by dougszumski — CHANGES_REQUESTED** [review]
Pending some questions on the upstream review

File: monasca_persister/conf/influxdb.py (hunk 1 above)
Comment (brtkwr) [reply monasca_persister/conf/influxdb.py]: Addressed

File: releasenotes/notes/influxdb-default-retention-policy-aaa446ebe6fc3cb5.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Support default_retention_policy config for InfluxDB for new
+    database that are created. Falls back to infinite retention as
+    before when unspecified. Only applicable when db_per_tenant is
+    enabled. Retention policy for existing databases are not altered.
+    Must be specified in format such as 1w, 1m, 1y, etc.
```
Comment (dougszumski) [inline releasenotes/notes/influxdb-default-retention-policy-aaa446ebe6fc3cb5.yaml]: This is no longer true - but it doesn't really matter apart from upstream

**Review by dougszumski — APPROVED** [review]
This looks sane - there is more work to be done upstream, but I think this should be compatible with how the final patch looks.

File: releasenotes/notes/influxdb-default-retention-policy-aaa446ebe6fc3cb5.yaml (hunk 2 above)
Comment (brtkwr) [reply releasenotes/notes/influxdb-default-retention-policy-aaa446ebe6fc3cb5.yaml]: Oops! Fixed.

**Pull request merged**